staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -1512,3 +1517,68 @@ message ReopenContainerLogRequest {
 
 message ReopenContainerLogResponse{
 }
+
+// Common options used for checkpointing and restoring.
+message CheckpointRestoreOptions {
```

Comment:
My preference is to add as few options as we can get away with. We could choose
reasonable defaults for the rest. Do we need anything besides the archive_location 
to begin with? We can always add options later if needed.

Reply:
I removed everything beside `archive_location` and `archive_is_oci_image` to easily distinguish if the destination is a local image or if should be pushed to a registry.

Reply:
I think that we can have it be local and have a path configured in the runtime as the default. The transportation of the checkpointed archive to a registry is a separate concern out of the CRI, so we don't need that. It could be handled by some operator later on.

Reply:
Also removed `archive_is_oci_image`.